_sass/_uswds-custom-styles.scss

```diff
@@ -219,6 +230,19 @@ $custom-image-dir: "../../img";
 // ==========================================================================
 // Footer
 // ==========================================================================
+.usa-footer {
```

Comment:
6. Updated the size of the logo in footer and link color of Contact Center

[Continue 👉]